dataframe-geo/build.gradle.kts

```diff
@@ -26,7 +26,8 @@ repositories {
 fun ExternalModuleDependency.excludeJaiCore() = exclude("javax.media", "jai_core")
 
 dependencies {
-    api(project(":core"))
+    api(projects.core)
+    implementation(projects.dataframeJupyter)
```

Comment:
Why do we need geo to depend on impl jupyter?

Reply:
Actually, yes, it uses `useSchema<>()` in its jupyter integration

Reply:
If we expect to use geo Integration to work only when jupyter Integration is configured (which should be), dependency can be `compileOnly` here

Reply:
I'm not 100% sure, @AndreiKingsley ?

Reply:
Let's fix this after merging, so we can revert back to java 8 sooner